.github/workflows/release-publish.yml

```
@@ -161,3 +161,45 @@ jobs:
         file: utils/automation/Dockerfile.ci
         labels: |
           org.opencontainers.image.version=${{ steps.get_release.outputs.tag_name }}.0
+
+  nomossa-build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      - name: Install dependencies
+        run: |
+          sudo apt-get update
+          sudo apt-get install -y \
+          build-essential pkg-config \
+          libglib2.0-dev librpm-dev \
+          libxml2-dev libxslt1-dev \
+          libjson-c-dev
+
+      - name: Setup CMake
+        uses: lukka/get-cmake@v4.0.3
+
+      - name: Build Nomossa
+        run: |
+          mkdir -p nomossa-build
+          cmake ./src/nomos -DCMAKE_BUILD_TYPE=MinSizeRel -B nomossa-build
+          cmake --build nomossa-build --target nomossa
```

```suggestion
          cmake --build nomossa-build --target nomossa
```
ninja -C ./nomossa-build/ nomossa you can build with this command